homeassistant/components/ovos/__init__.py

```diff
@@ -0,0 +1,44 @@
+"""The ovos integration."""
+from __future__ import annotations
+
+import logging
+
+from ovos_bus_client import MessageBusClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import discovery
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS: list[Platform] = [Platform.NOTIFY]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up ovos from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+    hass.data[DOMAIN].setdefault("entries", {})
```

Comment:
No need to store entries in a separate field `hass.data[DOMAIN][entry_id]` would be good enough